_templates/layout.html

```diff
@@ -52,6 +52,7 @@
         'audience_manager',
         'consent_manager',
         'tag_manager',
+        'container_settings',
```

Comment:
Please refrain from making changes here, unless you are adding stuff to the main menu in the top. This change is now messing it up.

Reply:
Fixed